resources/axon.rb

```diff
@@ -58,9 +58,30 @@
 
   # Set platform specific variables
   case node['platform']
-  when 'centos', 'redhat', 'suse', 'oraclelinux', 'oracle', 'amazon'
+  when 'centos', 'redhat', 'suse', 'oraclelinux', 'amazon'
     ext = '.rpm'
     eg_service_name = 'tw-eg-service'
+  when 'oracle'
+    ext = '.rpm'
+    eg_service_name = 'tw-eg-service'
+    # for Oracle UEK, need to run few extra commands as pre-requisites
+    if node["hostnamectl"]["kernel"].include?('uek')
+      if node['platform_version'] .include?('7.')
```

Comment:
instead of `include?`, a more precise strategy would be something like
```rb
case node['platform_version'].split('.')[0]
when '7'
# ...
else
# ...
end
```

Reply:
Yep. I concur.  They might end up doing a 8.7.1 which could break this.

Reply:
done